plone/app/querystring/tests/registry_testdata.py

```diff
@@ -1,66 +1,61 @@
-                                       u'created',
-                        'vocabulary': None,
-                        'title': u'Creation Date',
-                        'enabled': True,
-                        'sortable': False,
-                        'fetch_vocabulary': True
-                    }
+                        "group": "Dates",
+                        "description": "The time and date an item was " "created",
+                        "vocabulary": None,
+                        "title": "Creation Date",
+                        "enabled": True,
+                        "sortable": False,
+                        "fetch_vocabulary": True,
+                    },
                 },
-                'operation': {
-                    'date': {
-                        'largerThan': {
-                            'widget': None,
-                            'operation': u'plone.app.querystring.queryparser'
-                                         u'._largerThan',
-                            'description': u'Please use YYYY/MM/DD.',
-                            'title': u'after'
+                "operation": {
+                    "date": {
+                        "largerThan": {
+                            "widget": None,
+                            "operation": "plone.app.querystring.queryparser"
+                            "._largerThan",
+                            "description": "Please use YYYY/MM/DD.",
+                            "title": "after",
+                        },
+                        "lessThan": {
+                            "widget": None,
+                            "operation": "plone.app.querystring.queryparser."
+                            "_lessThan",
+                            "description": "Please use YYYY/MM/DD.",
+                            "title": "before",
                         },
-                        'lessThan': {
-                            'widget': None,
-                            'operation': u'plone.app.querystring.queryparser.'
-                                         u'_lessThan',
-                            'description': u'Please use YYYY/MM/DD.',
-                            'title': u'before'
-                        }
                     },
-                    'string': {
-                        'is': {
-                            'widget': None,
-                            'operation': u'plone.app.querystring.queryparser.'
-                                         u'_equal',
-                            'description': u'Tip: you can use * to '
-                                           u'autocomplete.',
-                            'title': u'equals'
+                    "string": {
+                        "is": {
+                            "widget": None,
+                            "operation": "plone.app.querystring.queryparser." "_equal",
+                            "description": "Tip: you can use * to " "autocomplete.",
```

Comment:
That's a bit annoying, but `black` wants to preserve the AST as a way to verify that it did not break anything, but then it leaves these strings that could be join together 
```suggestion
                            "operation": "plone.app.querystring.queryparser._equal",
                            "description": "Tip: you can use * to autocomplete.",
```